test/sass/typograpy.spec.scss

```diff
@@ -159,3 +164,118 @@
   }
 
 }
+
+@include describe('@mixin label-as-content') {
+
+  @include it('generates defaults for label-as-content') {
+
+    @include assert() {
+
+      @include output {
+        @include typography.label-as-content();
+      }
+
+      @include expect {
+        color: $text-secondary;
+        letter-spacing: 0.5px;
+        font-family: "Noto Sans", Arial, Helvetica, sans-serif;
+        font-size: 0.6875rem;
+        font-weight: 400;
+        line-height: 2.1818181818;
+        text-transform: uppercase;
+      }
+
+    }
+
+  }
+
+  @include it('generates different color for text and default text-transform') {
+
+    @include assert() {
+
+      @include output {
+        @include typography.label-as-content($primary);
+      }
+
+      @include expect {
+        color: $primary;
+        letter-spacing: 0.5px;
+        font-family: "Noto Sans", Arial, Helvetica, sans-serif;
+        font-size: 0.6875rem;
+        font-weight: 400;
+        line-height: 2.1818181818;
+        text-transform: uppercase;
+      }
+
+    }
+
+  }
+
+  @include it('generates default color and the text-transform is false') {
+
+    @include assert() {
+
+      @include output {
+        @include typography.label-as-content($uppercase: false);
+      }
+
+      @include expect {
+        color: $text-secondary;
+        letter-spacing: 0.5px;
+        font-family: "Noto Sans", Arial, Helvetica, sans-serif;
+        font-size: 0.6875rem;
+        font-weight: 400;
+        line-height: 2.1818181818;
+      }
+
+    }
+
+  }
+
+  @include it('generates different color and the text-transform is false') {
+
+    @include assert() {
+
+      @include output {
+        @include typography.label-as-content($color: $primary, $uppercase: false);
```

Comment:
Use local variable for the colour here, scoped to the `assert`, as described in a comment for a previous test in this file.